samples/mesh/Makefile

```diff
@@ -1,6 +1,6 @@
 XBE_TITLE = mesh
 GEN_XISO = $(XBE_TITLE).iso
-SRCS = $(wildcard $(CURDIR)/*.c)
+SRCS += $(CURDIR)/math3d.c $(CURDIR)/main.c
```

Comment:
Why the change to `+=`?

Reply:
You pushed since this review, but this one isn't fixed yet? @Teufelchen1